galasa-parent/dev.galasa.framework.api.authentication/src/main/java/dev/galasa/framework/api/authentication/internal/routes/AuthTokensRoute.java

```diff
@@ -233,4 +279,31 @@ private void addTokenToAuthStore(String clientId, String jwt, String description
         }
         logger.info("Stored token record in the auth store OK");
     }
+
+    private List<AuthToken> convertAuthStoreTokenIntoTokenBean(List<IInternalAuthToken> authStoreTokens){
```

Comment:
Missing an 's' in `convertAuthStoreToken` - should be something like `convertAuthStoreTokensIntoTokenBeans` since we're converting a list of tokens